scripts/run_client_tests_for_workspace.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# https://betterdev.blog/minimal-safe-bash-script-template/
```

Comment:
See just as a general rule for these scripts, I don't think we need to keep adding this boilerplate stuff. I've been writing Bash scripts for nearly ten years and never really found the need for this stuff.

Reply:
Okay, I've not, so not really sure. Happy to cede to your wisdom. We can do a follow up PR to clean it out in general if you like?